src/SolutionTemplate/Uno.ProjectTemplates.Dotnet/content/unoapp/.vsconfig

```diff
@@ -0,0 +1,112 @@
+{
+  "version": "1.0",
```

Comment:
With 'dotnet new' there's the option to exclude unwanted heads. Is the user going to get a message saying they need to install eg Xamarin.Android even if they've excluded the Android head?

Reply:
Most likely, but it's not possible to choose between iOS and Android, for instance. They can also ignore the message.